admin/tool/lp/classes/external.php

```diff
@@ -873,6 +873,8 @@ public static function search_users($query, $capability = '', $limitfrom = 0, $l
 
         $context = context_system::instance();
         self::validate_context($context);
+        require_capability('moodle/competency:templatemanage', $context);
+
```

Comment:
Just be careful with ``require_capability``.
I don't remember exactly what the issue was. But it has some to do with require login or something.

If you want to check capability, you might just need to use ``has_capability``

Reply:
This is from one of the tracker issues (MDL-83921)
It seems to work alright after implementing it to